### DISCUSSION TITLE
The password is invalid

### DISCUSSION THREAD

**Discussion by Gpan666** [LycheeOrg/Lychee#2755] [Bugs] [answered]
### Lychee version

v6

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

My password expired after updating from Lychee5 to Lychee6 and I was wondering if there was any way to change the password while keeping the data

### Diagnostics [REQUIRED]

![Uploading QQ20241128-020127.png…]()

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
If you know your username and have access to command line, you can do:

```
php artisan lychee:update_user <username> <password>
```
That will allow you to reset the password for given username.
